sdcm/microbenchmarking.py

```diff
@@ -97,7 +97,12 @@ def get_commit_id(x):
             last_val = get_metrica_val(list_of_results_from_db[last_idx])
             last_commit = get_commit_id(list_of_results_from_db[last_idx])
             cur_val = float(current_result["results"]["stats"][metrica])
-            min_result = min(list_of_results_from_db, key=get_metrica_val)
+
+            if 'frag/s' in metrica:
+                min_result = max(list_of_results_from_db, key=get_metrica_val)
```

Comment:
min_result name isn't good, we should change it to something like best_result

Reply:
done